<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>KeySequenceEditDialog</class>
 <widget class="QDialog" name="KeySequenceEditDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>664</width>
    <height>81</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QLineEdit" name="edtKeySequence">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <pointsize>24</pointsize>
      </font>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
     <property name="placeholderText">
      <string>Press keys...</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="checo::ui::MultiIconButton" name="btnOk">
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="icon">
      <iconset theme="tick_inactive"/>
     </property>
     <property name="iconSize">
      <size>
       <width>32</width>
       <height>32</height>
      </size>
     </property>
     <property name="autoDefault">
      <bool>false</bool>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="checo::ui::MultiIconButton" name="btnRestart">
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="icon">
      <iconset theme="restart_inactive"/>
     </property>
     <property name="iconSize">
      <size>
       <width>32</width>
       <height>32</height>
      </size>
     </property>
     <property name="autoDefault">
      <bool>false</bool>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="checo::ui::MultiIconButton" name="btnCancel">
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="icon">
      <iconset theme="cancel"/>
     </property>
     <property name="iconSize">
      <size>
       <width>32</width>
       <height>32</height>
      </size>
     </property>
     <property name="autoDefault">
      <bool>false</bool>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>checo::ui::MultiIconButton</class>
   <extends>QPushButton</extends>
   <header>checo/ui/multi_icon_button.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
